### PR TITLE
#166185724 Python coding style warnings/errors

### PR DESCRIPTION
**Pull request by reifred**
#### What does this PR do?
* Resolve formatting errors encountered on running flake8
#### Description of Task to be completed?
* Install flake8
* Format current files to remove bad formatting, incorrect styling, etc
#### How should this be manually tested?
* Install additional packages using pip install -r requirements_devel.txt
* Run ```flake8 wger``` to check the existence of any flake8 errors 
#### What are the relevant pivotal tracker stories?
[#166185724](https://www.pivotaltracker.com/story/show/166185724)

### CODE REVIEW

**Comment by Manorlds-Eaglespark** [comment]
Good work. I like how you described this PR. Am waiting till the tests run on Travis to approve.

**Comment by Emmanuel-Dominic** [comment]
@reifred, PR proves fake8 error fixation in the files that are updated to, so good work and thanks for the directions on `How should this be manually tested?`.

File: setup.cfg [hunk 1]
```diff
@@ -5,3 +5,8 @@ universal = 1
 exclude = urls.py,tasks.py,*migrations*,*bower_components*
 max-line-length = 100
 ignore = W503
+
+[flake8]
+exclude = urls.py,tasks.py,*migrations*,*bower_components*,*tests*
+max-line-length = 250
```
Comment (Rhytah) [inline setup.cfg]: Great work on implementing flake8 @reifred this line length is too long and could potentially lead to maintainability issues, as @Emmanuel-Dominic pointed out consider changing it

Comment (reifred) [reply setup.cfg]: Thanks for the feedback am going to work on it. I had implemented feedback got from you and @Emmanuel-Dominic.
image

**Review by Manorlds-Eaglespark — APPROVED** [review]

**Review by Emmanuel-Dominic — APPROVED** [review]

**Review by Rhytah — APPROVED** [review]
@reifred well done and the suggested changes to `max-line-length` were made.

**Pull request merged**